.github/workflows/publish-pypi.yaml

```
@@ -0,0 +1,103 @@
+# .github/workflows/publish-pypi.yaml
+# uses trusted publishing to publish the package to PyPI and create a GitHub
+# release as described here:
+# https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/
+name: Publish to PyPI and create GitHub release
+
+on:
+  push:
+    tags:
+      # only run workflow for tags in release format
+      - "v[0-9].[0-9].[0-9]"
```

I admire your goal of having no more than 10 patch versions per minor version (and subsequently 10 minor versions per major version)

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+"
```

---

😂 Great catch--thank you!